Makefile

```diff
@@ -10,7 +10,7 @@ NAMESPACE := $(or ${NAMESPACE},assisted-installer)
 GIT_REVISION := $(shell git rev-parse HEAD)
 PUBLISH_TAG := $(or ${GIT_REVISION})
 
-all: image image_controller image_controller_ocp unit-test
+all: build-images unit-test
 
 lint:
```

Comment:
I think `lint` can include `format-check`. I don't see anyone calling it directly. Most of the time you would like one target to verify everything.

Reply:
We settled this, I believe, in our discussion offline and the openshift/release PR. I went ahead and added both `lint` and `format-check` to the `all` target so, by default, everything gets checked.